css-basic/css-positioning_layout-with-position/css/styles.css

```diff
@@ -0,0 +1,116 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  margin: 80px 0 45px 0;
+  font-family: system-ui;
+}
+
+/**
+ *
+ * Place the header at the top of the viewport and make sure, that it remains there when the
+ * document is scrolled.
+ */
+.header {
```

Comment:
I added here a z-index: 2; to prevent some overlap between card-borders and header text when scrolling around. But I guess this is just a matter of taste. :)

Reply:
Ok, thanks!